### PR TITLE
[test_platform_info] Add check for return value of PsuController.get_psu_status as it could return empty list

### PR DESCRIPTION
**Pull request by Junchao-Mellanox**
### Description of PR

Summary:
Add check for return value of PsuController.get_psu_status as it could return empty list

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

#### What is the motivation for this PR?

PsuController.get_psu_status could return empty list. If we continue the test, it might mislead test case results reviewer because test case could failed at other place if `get_psu_status ` return empty.

#### How did you do it?

Add an assertion after get_psu_status

#### How did you verify/test it?

Manual test

#### Any platform specific information?

N/A

#### Supported testbed topology if it's a new test case?

N/A

### CODE REVIEW

File: tests/platform_tests/test_platform_info.py [hunk 1]
```diff
@@ -360,6 +361,7 @@ def test_thermal_control_psu_absence(duthosts, rand_one_dut_hostname, psu_contro
 
         logging.info('Shutdown first PSU and check thermal control result...')
         all_psu_status = psu_ctrl.get_psu_status()
+        assert all_psu_status and len(all_psu_status) >= 2, 'Failed to get at least 2 PSU status: {}'.format(all_psu_status)
```
Comment (wangxin) [inline tests/platform_tests/test_platform_info.py:364]: LGTM. Just one minor comment: 
Can you replace the raw `assert` with wrapper `pytest_assert` defined in `tests.common.helpers.assertions.py`?

Comment (Junchao-Mellanox) [reply tests/platform_tests/test_platform_info.py:364]: Sure.

**Comment by liat-grozovik** [comment]
@Junchao-Mellanox I see you fixed the assert to python_assert as recommended on your recent changes but I would suggest to replace all the asserts in the code with this one.

**Comment by Junchao-Mellanox** [comment]
> @Junchao-Mellanox I see you fixed the assert to python_assert as recommended on your recent changes but I would suggest to replace all the asserts in the code with this one.

I will change all assert to pytest_assert in test_platform.info.py.

File: tests/platform_tests/test_platform_info.py (hunk 1 above)
Comment (yxieca) [reply tests/platform_tests/test_platform_info.py]: If there is not enough PSU len(..) < 2, do you want to fail the test or skip the test.

Maybe pytest_requires() is a better choice than pytest_assert?

**Review by yxieca — APPROVED** [review]

**Review by wangxin — APPROVED** [review]

**Pull request merged**